pandas/tests/indexes/test_setops.py

```diff
@@ -0,0 +1,72 @@
+'''
+The tests in this package are to ensure the proper resultant dtypes of
+set operations.
+'''
+import itertools as it
+
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.common import is_dtype_equal
+
+import pandas as pd
+from pandas import Int64Index, RangeIndex
+from pandas.tests.indexes.conftest import indices
```

Comment:
you don't import, just use it (it will be picked up automatically)

Reply:
The import here is because I need to access the indices to create the pair combinations for the `test_union_different_types` test. See the parametrize decorator for how this is being used on line 34.

Reply:
ok, rather than import the fixture directly, make the fixture (in pandas/tests/indexes/conftest.py) more like

```
_indices_list = [tm.makeIndex(....), .......]

@pytest.mark.parametrize(params=_indices_list, ids=.......)
def indices_fixture(...)
    .....
```

I made a similar suggestion in #23943 FYI

Reply:
done.